tests/test_metric_store.py

```diff
@@ -24,14 +23,6 @@ def metrics_sample():
     }
 
 
-def test_null_object_metrics_store_always_return_true(metrics_sample):
```

Comment:
We [agreed with @Robert-Steiner](https://github.com/xainag/xain-fl/pull/244#discussion_r374197388) in #244 that the method for writing metrics would use exceptions instead of returning a boolean, to signal errors. @Robert-Steiner did the changes, but this test was forgotten and CI did not complain at the time. But now, this test is failing, so I just removed it.